svelte-cloudinary/src/routes/types.ts

```diff
@@ -0,0 +1,23 @@
+type BooleanStringEffect = 'pixelate' | 'blur' | 'cartoonify' | 'gradientFade' | 'sharpen';
```

Comment:
There might be a better way to do all this typing by referencing the component's types directly, but I wasn't totally sure about that.